### PR TITLE
Battle background image

### PR DESCRIPTION
**Pull request by JotinhaPorto**
https://github.com/user-attachments/assets/6b9973df-af3e-42dc-be42-857a1634affe

### CODE REVIEW

File: src/components/BattleBackgroundImage.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+type Props = { children: React.ReactNode };
+const BattleBackgroundImage = ({ children }: Props) => {
+  return (
+    <div className='bg-[url("./public/images/bgDesert.png")] absolute w-full h-full bg-no-repeat bg-cover flex flex-col justify-center '>
```
Comment (avilabiel) [inline src/components/BattleBackgroundImage.tsx]: Faz a imagem ficar centralizada no BG. Vai ser útil pra mostrar o castelo no mobile

Comment (avilabiel) [reply src/components/BattleBackgroundImage.tsx]: Ref: https://tailwindcss.com/docs/background-position

**Review by avilabiel — COMMENTED** [review]
Por favor, subir as imagens no TinyPNG

File: src/components/BattleBackgroundImage.tsx (hunk 1 above)
Comment (JotinhaPorto) [reply src/components/BattleBackgroundImage.tsx]: image

**Review by avilabiel — APPROVED** [review]

**Pull request merged**